components/style/properties/properties.mako.rs

```diff
@@ -2048,6 +2048,22 @@ pub fn modify_style_for_anonymous_table_object(
     let box_style = Arc::make_mut(&mut style.box_);
     box_style.display = new_display_value;
     box_style.position = longhands::position::computed_value::T::static_;
+
+    let padding = Arc::make_mut(&mut style.padding);
```

Comment:
It’s not a question of "visual" style (which is not a thing), but whether a given property is *inherited* (like `font-size`) or not (like `margin-top`).

https://drafts.csswg.org/css2/visuren.html#anonymous-block-level
> The properties of anonymous boxes are inherited from the enclosing non-anonymous box (e.g., in the example just below the subsection heading "Anonymous block boxes", the one for DIV). Non-inherited properties have their initial value. For example, the font of the anonymous box is inherited from the DIV, but the margins will be 0.